dev/tasks/python-wheels/github.linux.amd64.yml

```diff
@@ -29,18 +29,9 @@ jobs:
     name: "Build wheel for Manylinux {{ manylinux_version }}"
     runs-on: ubuntu-latest
     env:
-      # archery uses this environment variable
+      # archery uses these environment variables
+      ARCH: amd64
       PYTHON: {{ python_version }}
-      # this is a private repository at the moment (mostly because of licensing
-      # consideration of windows images with visual studio), but anyone can
-      # recreate the image by manually building it via:
-      # `archery build python-wheel-manylinux-2010|2014`
-      # note that we don't run docker build since there wouldn't be a cache hit
-      # and rebuilding the dependencies takes a fair amount of time
-      # (in case of linux multi-host caching works much better with buildkit)
-      REPO: ghcr.io/ursacomputing/arrow
```

Comment:
Changed it to use the more widely accessible apache dockerhub repo (originally changed to use a private repo because of the windows images).